README.md

```diff
@@ -6,6 +6,7 @@ A project based learning activity for people who are getting started with Git an
 
 1. Press the space bar to begin.
 2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
```

Comment:
Sometimes I respond too fast for the page to update! If you perform an expected action and don't see a response, wait a few seconds and refresh the page for your next steps.